kolibri/utils/cli.py

```diff
@@ -62,6 +62,7 @@
   kolibri url               Tell me the address of Kolibri
   kolibri shell             Display a Django shell
   kolibri manage help       Show the Django management usage dialogue
+  kolibri manage shell_plus Preload all models in Django shell
```

Comment:
It's not nice to load our CLI usage instructions any further than they already are. `shell_plus` is a developer tool that some developers may like, but only some, so basically I don't think it's warranted in the CLI.